app/controllers/links_controller.rb

```diff
@@ -23,9 +23,12 @@ def create
     end
 
     if @link.save
-      redirect_to link_path(@link.short_url), notice: "URL added"
+      #redirect_to link_path(@link.short_url), notice: "URL added"
+      redirect_to root_url, notice: "URL added"
     else
-      render root_url, error: "Your URL was not valid"
+      #render root_url, error: "Your URL was not valid"
```

Comment:
OK...cleanup in progress

Code commit coming soon..

On Thu, Apr 9, 2015 at 1:09 PM, Ben Woodall <notifications@github.com>
wrote:

> In app/controllers/links_controller.rb
> <https://github.com/benwoody/shrtnr/pull/4#discussion_r28097778>:
>
> >      else
> > -      render root_url, error: "Your URL was not valid"
> > +      #render root_url, error: "Your URL was not valid"
>
> Nice change! Make sure you clean up the comments before you commit [image:
> :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/4/files#r28097778>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw